src/Sulu/Bundle/AdminBundle/Metadata/SchemaMetadata/SchemaMetadata.php

```diff
@@ -59,7 +68,9 @@ public function toJsonSchema(): array
          * empty schema object as array instead of an object and would break
          */
         if (empty($jsonSchema)) {
-            $jsonSchema['required'] = [];
+            return [
+                'type' => ['number', 'string', 'boolean', 'object', 'array', 'null'],
```

Comment:
This schema matches every value, the same as `['required' => []]` did, but doesn't throw a warning because no type was specified

Reply:
not perfectly sue, but i think we can use `return new \stdClass();` to return an empty schema that will be serialized to an empty object instead of an array. that would solve the problem described above and we dont need to hardcode all tpes here 🙂

Reply:
The `toJsonSchema()` method has `array` as return type, and I think I would not change that ... But these hard coded types won't change, because those are all valid json types

Reply:
you are right. this way is okay for me too 🙂